### PR TITLE
bower

### PR DESCRIPTION
**Pull request by epicfaace**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "chrome-tabs",
+  "homepage": "https://github.com/adamschwartz/chrome-tabs",
+  "authors": [
+    "Adam Schwartz"
+  ],
+  "description": "Chrome Tabs in Chrome",
+  "main": "js/chrometabs.js",
```
Comment (adamschwartz) [inline bower.json]: I believe this path is incorrect. Should be `js/chrome-tabs.js` I believe.

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "chrome-tabs",
+  "homepage": "https://github.com/adamschwartz/chrome-tabs",
+  "authors": [
+    "Adam Schwartz"
+  ],
+  "description": "Chrome Tabs in Chrome",
```
Comment (adamschwartz) [inline bower.json]: Please change description to `"Chrome-style tabs in HTML/CSS"`.

**Comment by epicfaace** [comment]
@adamschwartz Those changes have been implemented now.

**Comment by adamschwartz** [comment]
Thx!

**Comment by epicfaace** [comment]
It appears that "bower chrome-tabs" is already registered to this repository, though: https://github.com/zedesk/chrome-tabs

**Pull request merged**